Documentation/specs/corehost.md

```diff
@@ -0,0 +1,112 @@
+# corehost runtime/assembly resolution
+
+The shared host locates assemblies and native libraries using a combination of: Servicing Index, Files in the application folder (aka "app-local") and files from package caches.
+
+## Definitions and Formats
+
+### Servicing Index
+
+The servicing index is loaded when the `DOTNET_SERVICING` environment variable is non-empty. When this variable is non-empty, it points to a directory that will be the **Servicing Root**.
```

Comment:
On Windows, I believe we will have a default value (DNX did). Not sure about Linux/Mac, but I do agree that having a default might be good.